charts/gpu-metrics-exporter/values-gke.yaml

```diff
@@ -1,7 +0,0 @@
-affinity:
```

Comment:
This file is cleared out to but not deleted because the install script depends on the file being present. After we update the install script we can delete tis and the other values-*.yaml files